configure.ac

```
@@ -1819,14 +1818,19 @@ AS_CASE([$enable_systhreads,$enable_unix_lib],
         [systhread_support=false
         AC_MSG_NOTICE([the POSIX threads library is not supported])])])])])
 
-## Does the assembler support debug prefix map and CFI directives
+## Does the assembler support debug prefix map
 as_has_debug_prefix_map=false
+AS_IF([$native_compiler],
+  [AS_CASE([$host],
+    [*-pc-windows], [],
+    [OCAML_AS_HAS_DEBUG_PREFIX_MAP])])
```

```suggestion
AS_CASE([$native_compiler,$host],
  [*,*-pc-windows], [],dnl MSVC definitely doesn't support it
  [true,*], [OCAML_AS_HAS_DEBUG_PREFIX_MAP])
```

That said, as long as the test doesn't crash cl, we could just let the check run and have less m4sh in `configure.ac` 🤷‍♀️

---

Applied, thanks.